spec/ttml1.xml

```diff
@@ -5672,6 +5672,10 @@ each character child of a <el>p</el> element is considered to be enclosed in an
 span.</p>
 <p>The <att>tts:visibility</att> style has no affect on content layout or
 composition, but merely determines whether composed content is visible or not.</p>
+<p>If the computed value of the property associated with this attribute is visible, then areas generated
```

Comment:
The 1st and 5th instances of 'visible' should be wrapped in ``<code>...</code>``; likewise for the 1st instance of 'hidden'. This is to match the formatting in TTML2.